data/src/apiKey/java/org/cryptomator/data/cloud/onedrive/OnedriveCloudNodeFactory.kt

```diff
@@ -74,8 +74,8 @@ internal object OnedriveCloudNodeFactory {
 	}
 
 	private fun lastModified(item: DriveItem): Date? {
-		return item.lastModifiedDateTime?.let {
-			return Date.from(it.toInstant())
+		return item.fileSystemInfo?.let {
+			return Date.from(it.lastModifiedDateTime?.toInstant())
```

Comment:
`lastModifiedDateTime` is `@Nullable` as well.

Reply:
Hi @SailReal, this file already has the changes that you have requested. Maybe I’m missing something